circuits/circom/merkleTree.circom

```diff
@@ -0,0 +1,41 @@
+// Whole file copied from https://github.com/ChihChengLiang/poseidon-tornado
+// TODO: import as a dependency - note circom doesn't appear to work well with nested dependencies, as poseidon-tornado needs its own node_modules for circomlib
```

Comment:
This file is just a dependency, and will be moved to node_modules or a submodule shortly.